typos.toml

```diff
@@ -0,0 +1,46 @@
+[default.extend-words]
+# random words from unit tests
+Pn = "Pn"
+fo = "fo"
+nd = "nd"
+Nd = "Nd"
+ba = "ba"
+ECT = "ECT"
+Ue = "Ue"
+Iy = "Iy"
+hte = "hte"
+numer = "numer"
+abd = "abd"
+aroun = "aroun"
+abov = "abov"
+Ois = "Ois"
+alo = "alo"
+
+# abbreviations, common words, etc.
+typ = "typ"
+datas = "datas"
+YOUY = "YOUY"
+lits = "lits"
+
+# exposed to public API
+Serializeable = "Serializeable"
+
+# from test cases like TPC-* or ClickBench
+carefull = "carefull"
+precentage = "precentage"
```

Comment:
Can we remove these? These are clearly misspelled words that should be flagged.
If they show up in some test data we don't want / can't modify, maybe we can exclude-list that data set instead,